### PR TITLE
feat: support alternative package managers (yarn/pnpm)

### PR DESCRIPTION
**Pull request by ghost**
Added support for yarn and pnpm.

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -124,3 +126,45 @@ async function _replacePlaceholder(wdioConf: string) {
     fileString = fileString.replace(/%baseUrl%/g, BASE_URL)
     await fs.writeFile(wdioConf, fileString)
 }
+
+function _getUsedPackageManager(): PackageManager {
+    const alternativeManagers = [PackageManager.pnpm, PackageManager.pnpm]
```
Comment (vobu) [inline src/index.ts]: this should be `const alternativeManagers = [PackageManager.pnpm, PackageManager.yarn]`, right? :)

File: src/index.ts [hunk 2]
```diff
@@ -124,3 +126,45 @@ async function _replacePlaceholder(wdioConf: string) {
...
+    const defaultManager = PackageManager.npm
+
+    const { npm_config_user_agent } = process.env
```
Comment (vobu) [inline src/index.ts:134]: there is a `user agent` also for package managers, wow. cool, learned something new here!

**Review by vobu — CHANGES_REQUESTED** [review]
thanks a ton for this enhancement already!
gave it a quick spin locally, things seem to work fine (yeah, missing CI here,  I know...)
minor coding error found, then it should be good to go.

File: src/index.ts (hunk 2 above)
Comment (ghost) [reply src/index.ts:134]: Yeah, this was my first go-to, just logging process.env to see if there is something interesting. Fortunately there is :)

File: src/index.ts (hunk 1 above)
Comment (ghost) [reply src/index.ts]: Funny that it still worked when I tried `yarn create`. Maybe they are more compatible than `pnpm` and `npm`

**Review by vobu — APPROVED** [review]

**Pull request merged**